src/page-modules/departures/layout.tsx

```diff
@@ -207,9 +217,20 @@ function DeparturesLayout({
                 : t(PageText.Departures.search.searching.nearby)}
             </Typo.p>
           </motion.div>
-        ) : (
-          children
         )}
+
+        {!isSearching && geolocationError !== null && hasEmptyChild && (
+          <EmptyPageContent
```

Comment:
Should we show something similar for the assistant page? Different text of course, since what it says about departures don't match.